docs/user-guide/projects.md

```diff
@@ -321,8 +321,8 @@ stringData:
 
 All the examples above talk about Git repositories, but the same principles apply to clusters as well.
 
-With cluster-scoped clusters we can also restrict projects to only allow applications whose destinations belong to the 
 same project. The default behavior allows for applications to be installed onto clusters which are not a part of the same 
+With project-scoped clusters we can also restrict projects to only allow applications whose destinations belong to the
```

Comment:
Looks like the sentence has been moved down now, the only thing that should be substituted is `cluster-scoped` to `project-scoped`